svc-mesh/service-mesh.md

```diff
@@ -0,0 +1,251 @@
+```
+$ kubectl get svc -n istio-system
+$ kubectl get pods -n istio-system -o wide
+```
+
+### Conduit
+
+As a reference, you can follow Conduit [instructions](https://conduit.io/getting-started/index.html).
+
+Here is a summary of what you need to install Conduit on your system
+```
+$ curl https://run.conduit.io/install | sh
+$ export PATH=$PATH:$HOME/.conduit/bin
+```
+
+Let's now deploy Conduit in the control plane of your cluster
+```
+$ conduit install | kubectl apply -f -
+```
+
+In order to verify the control plane has been properly deployed, you
+can use both of those following commands
+```
+$ kubectl get svc -n conduit
+$ kubectl get pods -n conduit -o wide
+```
+
+## Inject your services with sidecars
+
+Once the control plane is running, you need a deployment defining a
+few services relying on each other. Then you need to inject the YAML
+file with the sidecar proxy, using the tools provided by each service
+mesh.
+
+If you don't have such a deployment ready, you can refer to the samples
+provided by each project.
+
+### Istio
+
+Istio provides a [bookinfo](https://istio.io/docs/guides/bookinfo/)
+sample on which you can rely on to inject their `envoy` proxy as a
+sidecar.
+
+You need to use their tool called `istioctl kube-inject` to inject
+your YAML file. We are taking their [bookinfo](https://istio.io/docs/guides/bookinfo/)
+sample as example
+```
+$ istioctl kube-inject -f samples/bookinfo/kube/bookinfo.yaml -o bookinfo-injected.yaml
+```
+
+### Conduit
+
+Conduit provides an [emojivoto](https://conduit.io/getting-started/index.html)
+sample on which you can rely on to inject their `conduit` proxy as
+a sidecar.
+
+You need to use their tool called `conduit inject` to inject your YAML
+file. We are taking their [emojivoto](https://conduit.io/getting-started/index.html)
+sample as example
+```
+$ wget https://raw.githubusercontent.com/runconduit/conduit-examples/master/emojivoto/emojivoto.yml
+$ conduit inject emojivoto.yml > emojivoto-injected.yaml
+```
+
+## Run your services with Kata
+
+Now that you have your service deployment injected with the appropriate
+sidecar containers, you need to manually edit your deployment to make it
+work with Kata.
+
+### Lower privileges
+
+In Kubernetes, the __init__ container is often `privileged` as it needs to
+setup the environment, which often needs some root privileges. But in case
+of those service meshes, all they need is the `NET_ADMIN` capability so that
+they can modify the underlying network rules. Conduit, by default, does not
+use `privileged` container, but Istio does.
+
+For this reason, if you are using Istio, you will need to switch all
+containers with `privileged: true` to `privileged: false`.
+
+### Add annotations
+
+This last section is the common part which does not make any difference
+between Istio and Conduit because it is about which CRI implementation
+you are using.
+
+For both CRI-O and CRI-containerd, you will have to add an annotation
```

Comment:
For both CRI-O and CRI-containerd, you have to add an annotation

Reply:
done